content/courses/cmpe230.tr.md

```diff
@@ -32,8 +32,6 @@ aliases:
 
 ## Catalog Description
```

Comment:
Titles left in English.

Reply:
Bu benim gözümden kaçmış şimdi issues'a bakarken fark ettim, hemen düzelttim çok özür dilerim